lib/errors.js

```diff
@@ -1,398 +1,3 @@
 'use strict';
 
-/**
```

Comment:
moved to lib/errors/index.js, to get a more reasonable separation of the classes in the api ref (subfolder = new section)